docs/welcome.md

```diff
@@ -1,14 +1,13 @@
 ---
 title: Welcome to .NET
 description: Getting started with the .NET family of technologies.
-keywords: .NET, .NET Core, getting started, news
 author: mairaw
 ms.author: mairaw
-ms.date: 08/14/2017
+ms.date: 04/30/2018
 ms.topic: article
-ms.prod: .net
+ms.prod: dotnet
 ms.devlang: dotnet
-ms.assetid: cb788dcf-2120-467f-9c34-c02a90e1f68f
+ms.custom: "updateeachrelease"
 ---
 # Welcome to .NET
 
```

Comment:
nit unrelated to your PR for line 14: The title is "Get Started with .NET Core", not "Getting...."